bluemira/geometry/parameterisations.py

```diff
@@ -195,17 +195,15 @@ def create_array(self, n_points=200, by_edges=True, d_l=None):
 
         Returns
         -------
-        xyz: np.ndarray
+        xyz: Coordinates
```

Comment:
Just wondering if this method might end up being confusing since it's called `create_array` but actually returns a `Coordinates` object, which looks like an array but has some different functionality from an array. More a point for discussion really, but perhaps what should be returned here is the underlying array rather than the coordinates?

Reply:
Hmm.. It is a little misleading. Thing is, I think we normally want coordinates, rather than arrays, seeing as the totality (as far as I have seen so far) of operations involve indexing rather than e.g. a dot product or cross product or whatever. `Coordinates` were intended to behave completely like `np.ndarray` but I found that it wasn't required, and would introduce a lot more work.

Would you be happy with `create_coordinates` as a method name?

Reply:
Perhaps we could call the method `discretize` since we're essentially wrapping the discretisation of the shape corresponding to that parameterisation?

Reply:
I guess it is also debatable that this method should exist at all. I know Ivan mentioned this in the past. The reason I left it in was to allow the possibility for somebody to override it by creating the coordinates directly without making the primitive shape. For instance this could be done for the `PrincetonD` by returning the original points, rather than making the original points into a primitive and then discretising the primitive. That said, we're not really using this at the moment. It does look like some premature optimisation... the idea was lets not make a primitive if we don't need to.